### PR TITLE
Securing Artificial Intelligence from Adversaries

### PR DESCRIPTION
**Pull request by collins-a**
Included article as index.md
hero image as hero.jpg
matrix image as matrix.jpg
boundary image as boundary.jpg
miscategorization image as miscategorization.png

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      

#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by collins-a** [comment]
This fixes #600

File: articles/securing-ai-systems-from-adversaries/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+##### Defense
+
+An effective way to defend AI from model duplicating techniques is the use of a privacy framework named Private Aggregation of Teacher Ensembles (PATE). PATE is motivated by the concerns about the privacy of sensitive data that is used to train several machine learning models. These models need to be prevented from revealing confidential details of the sensitive data.
+
+PATE works on the principle that involves training several models on disjoint data. These models are referred to as “teacher” models. If these models agree on input, no confidential details from their training set are leaked. The models agree since they all reach the same conclusion. For privacy to be achieved, the outputs of the aforementioned models need to be in consensus.
```
Comment (lalith1403) [inline articles/securing-ai-systems-from-adversaries/index.md:46]: What do you mean by these models agree on input? It's not clear how they reach the same conclusion. If the models are trained on dis-joint data, then how will they reach the same conclusion? Is it happening batch-wise?

File: articles/securing-ai-systems-from-adversaries/index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+Boundary shift [Source](https://arxiv.org/pdf/1904.06292.pdf)
+
+Here we look at a threat known as machine learning model skewing as an example of a data poisoning attack.
+
+##### Model Skewing
+
+In classification models, models skewing attacks aim to shift the classification boundary. The classification boundary separates what is considered as good input from bad input. This is illustrated by the Boundary Shift image above.
+
+##### Defense
```
Comment (lalith1403) [inline articles/securing-ai-systems-from-adversaries/index.md:96]: Is this defense for model skewing or poisoning attacks? Mention the same.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Very well written article. A few minor changes, and this should be ready for the final round of review. Cheers.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by collins-a** [comment]
Hey, @lalith1403 I've made the requested minor tweaks.  If they suffice, see you next article. Cheers!

**Pull request merged**